mne/epochs.py

```diff
@@ -809,14 +809,21 @@ def __next__(self, *args, **kwargs):
         """Provide a wrapper for Py3k."""
         return self.next(*args, **kwargs)
 
-    def average(self, picks=None):
-        """Compute average of epochs.
+    def average(self, picks=None, method="mean"):
+        """Compute an aggregate over epochs. Defaults to the mean.
 
         Parameters
         ----------
         picks : array-like of int | None
             If None only MEG, EEG, SEEG, ECoG, and fNIRS channels are kept
             otherwise the channels indices in picks are kept.
+        method : str | callable
+            How to combine the data. If "mean", "std" or "gfp", the mean,
+            standard deviation or global field power are returned.
+            Otherwise, must be a callable which, when passed an (a x b x c)
```

Comment:
an array of shape (n_epochs, n_channels, n_time) returns an array of shape (n_channels, n_time)

e.g.::

the double-colon makes it render as code